protocol/communities_messenger_test.go

```diff
@@ -75,18 +75,18 @@ func (s *MessengerCommunitiesSuite) TearDownTest() {
 	_ = s.logger.Sync()
 }
 
-func (s *MessengerCommunitiesSuite) newMessengerWithKey(privateKey *ecdsa.PrivateKey) *Messenger {
-	messenger, err := newCommunitiesTestMessenger(s.shh, privateKey, s.logger, nil, nil)
+func (s *MessengerCommunitiesSuite) newMessengerWithKey(privateKey *ecdsa.PrivateKey, password string, walletAddresses []string) *Messenger {
+	messenger, err := newCommunitiesTestMessenger(s.shh, privateKey, s.logger, &TokenManagerStub{}, password, walletAddresses)
```

Comment:
As shown here: https://github.com/status-im/status-go/pull/3709/files, I believe this API don't need password and addresses.

Reply:
Thanks! I cherry-picked your changes